modules/machineset-creating.adoc

```diff
@@ -122,6 +122,12 @@ $ oc get node <node_name> --show-labels
 Review the command output and confirm that `node-role.kubernetes.io/<your_label>`
 is in the `LABELS` list.
 
+[NOTE]
+====
+Labels edited or added to an existing MachineSet are not propogated to existing Machines and Nodes
+associated with the MachineSet.
+====
+
```

Comment:
Also, for [1], where an AWS instanceType is edited, the same holds for any other field in machineset. Once a machine owned by a machineset is created, any consequent change to the machineset takes no effect on the machine. What about:
```
Any change to a MachineSet takes no effect on already existing Machines owned by the MachineSet. E.g. labels edited or added to an existing MachineSet are not propagated to existing Machines and Nodes
associated with the MachineSet.
```

[1] https://bugzilla.redhat.com/show_bug.cgi?id=1745132